mypy/test/teststubgen.py

```diff
@@ -906,6 +931,54 @@ class TestClass(argparse.Action):
         assert_equal(output, ["class C(argparse.Action): ..."])
         assert_equal(gen.get_imports().splitlines(), ["import argparse"])
 
+    @unittest.skipIf(sys.version_info < (3, 12), "Inline Generics not supported before Python3.12")
+    def test_inline_generic_class(self) -> None:
+        T = TypeVar("T")
+
+        class TestClass:
+            __type_params__ = (T,)
+
+        output: list[str] = []
+        mod = ModuleType("module", "")
+        gen = InspectionStubGenerator(mod.__name__, known_modules=[mod.__name__], module=mod)
+        gen.generate_class_stub("C", TestClass, output)
+        assert_equal(output, ["class C[T]: ..."])
+
+    @unittest.skipIf(sys.version_info < (3, 12), "Inline Generics not supported before Python3.12")
+    def test_generic_class(self) -> None:
+        exec("class Test[A]: ...")
```

Comment:
you forgot the `gen.generate_class_stub` part :)

Reply:
Oh I think I now get what you mean. I was quite confused my apologies.